pkg/cedar/core/authority_to_operate.go

```diff
@@ -0,0 +1,126 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/guregu/null"
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	apiauthority "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/authority_to_operate"
+)
+
+// GetAuthorityToOperateOptionalParams represents the optional parameters that can be used to filter ATO information when searching through the CEDAR API
+type GetAuthorityToOperateOptionalParams struct {
+	CedarSystemID                             null.String
+	ContainsPersonallyIdentifiableInformation null.Bool
+	DispositionDateAfter                      *time.Time
+	DispositionDateBefore                     *time.Time
+	FismaSystemAcronym                        null.String
+	IsProtectedHealthInformation              null.Bool
+	SystemID                                  null.String
+	TlcPhase                                  null.String
+	UUID                                      null.String
+}
+
+// GetAuthorityToOperate makes a GET call to the /authority_to_operate endpoint
+func (c *Client) GetAuthorityToOperate(ctx context.Context, optionalParams *GetAuthorityToOperateOptionalParams) ([]*models.CedarAuthorityToOperate, error) {
```

Comment:
As mentioned above in my comment on `GetAuthorityToOperateOptionalParams` and the thread with Clay, we probably want this method to take a single, required string parameter `cedarSystemID` (the version-agnostic system ID).

Reply:
Implemented as of https://github.com/CMSgov/easi-app/pull/1636/commits/b421c6bbac17cdbb5fb9086448d5bcce549eb523.